lua/framework/physics/lua_material.h

```diff
@@ -115,11 +115,19 @@ int MatAddProperty(lua_State* L);
  * ####_
  *
  * OPENSN_XSFILE\n
- * Loads transport cross-sections from OpenSn cross-section files. Expects
+ * Loads transport cross sections from OpenSn cross-section files. Expects
```

Comment:
```suggestion
 * Loads transport cross sections from OpenSn cross section files. Expects
```

Reply:
Fixed throughout the code base. Hopefully...